package.json

```diff
@@ -1,121 +1,56 @@
-      "import": "./dist/operation/get-parameters-as-json-schema.js"
-    },
-    "./package.json": "./package.json",
-    "./utils": {
-      "require": "./dist/utils.cjs",
-      "import": "./dist/utils.js"
-    }
-  },
-  "main": "dist/index.cjs",
-  "module": "dist/index.js",
-  "types": "dist/index.d.cts",
-  "engines": {
-    "node": ">=18"
-  },
-  "files": [
-    "dist"
-  ],
-  "keywords": [
-    "api",
-    "apis",
-    "openapi",
-    "openapi initiative",
-    "openapi specification",
-    "openapi spec",
-    "openapi document",
-    "oai",
-    "oas",
-    "apidoc",
-    "microservice",
-    "documentation"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/readmeio/oas.git"
-  },
-  "bugs": {
-    "url": "https://github.com/readmeio/oas/issues"
-  },
+  "private": true,
   "scripts": {
-    "build": "tsup",
-    "lint": "npm run lint:types && npm run lint:js && npm run lint:docs && npm run prettier",
-    "lint:js": "eslint . --ext .js,.ts",
-    "lint:docs": "npx -y alex .",
-    "lint:types": "tsc --noEmit",
-    "prebuild": "rm -rf dist/",
-    "prelint": "npm run build",
-    "prepack": "npm run build",
+    "alex": "alex .",
+    "build": "npx lerna run build --stream",
+    "clean": "npx lerna clean",
+    "lint": "npm run lint:types && npm run lint:js && npm run prettier",
+    "lint:js": "npm run lint:js --if-present --workspaces",
+    "lint:types": "npm run lint:types --if-present --workspaces",
+    "prelint": "npm run prelint --if-present --workspaces",
     "prepare": "husky install",
-    "pretest": "npm run build",
     "prettier": "prettier --check .",
-    "prettier:fix": "prettier --check --write .",
-    "test": "vitest --coverage",
-    "watch": "tsc --watch"
+    "prettier:write": "prettier --check --write .",
+    "publish": "npx lerna publish",
+    "pretest": "npm run build",
+    "test": "npx lerna run test --stream"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/readmeio/oas.git"
   },
-  "dependencies": {
-    "@readme/json-schema-ref-parser": "^1.2.0",
-    "@types/json-schema": "^7.0.11",
-    "json-schema-merge-allof": "^0.8.1",
-    "jsonpath-plus": "^7.2.0",
-    "jsonpointer": "^5.0.0",
-    "memoizee": "^0.4.14",
-    "oas-normalize": "^11.0.0",
-    "openapi-types": "^12.1.1",
-    "path-to-regexp": "^6.2.0",
-    "remove-undefined-objects": "^4.0.2"
+  "engines": {
+    "node": ">=18"
   },
+  "workspaces": [
+    "./packages/*"
+  ],
   "devDependencies": {
-    "@commitlint/cli": "^17.7.1",
+    "@commitlint/cli": "^17.6.6",
     "@commitlint/config-conventional": "^17.7.0",
-    "@readme/eslint-config": "^13.0.1",
-    "@readme/oas-examples": "^5.12.0",
-    "@readme/openapi-parser": "^2.5.0",
-    "@types/json-schema-merge-allof": "^0.6.1",
-    "@types/memoizee": "^0.4.6",
-    "@types/node": "^20.5.9",
-    "@vitest/coverage-v8": "^0.34.4",
+    "@readme/eslint-config": "^12.2.1",
```

Comment:
why's this v12?

Reply:
not sure but i fixed this in https://github.com/readmeio/oas/pull/811.